addons/html_builder/static/src/builder/plugins/overlay_buttons/overlay_buttons.xml

```diff
@@ -0,0 +1,13 @@
+<templates xml:space="preserve">
+
+<t t-name="html_builder.OverlayButtons">
+    <div t-if="state.isVisible" class="o_overlay_options">
+        <t t-foreach="state.buttons" t-as="button" t-key="button_index">
+                <button t-att-class="button.class" t-on-click="button.event === 'click' ? button.handler : () => {}"/>
```

Comment:
it s possible to have a button without handler ?

Reply:
an exemple for  button.event !== click ?

Reply:
It was to anticipate for the drag button later, but let's not do it now indeed :slightly_smiling_face: